playbooks/init/vars/cluster_hosts.yml

```diff
@@ -22,5 +22,6 @@ g_glusterfs_registry_hosts: "{{ groups.glusterfs_registry | default(g_glusterfs_
 g_all_hosts: "{{ g_master_hosts | union(g_node_hosts) | union(g_etcd_hosts)
                  | union(g_new_etcd_hosts) | union(g_lb_hosts) | union(g_nfs_hosts)
                  | union(g_new_node_hosts)| union(g_new_master_hosts)
-                 | union(g_glusterfs_hosts) | union(g_glusterfs_registry_hosts)
+                 | union(g_glusterfs_hosts if openshift_storage_glusterfs_is_native is defined and not openshift_storage_glusterfs_is_native else [])
```

Comment:
Can you explain this change?  I don't see why we should do this?

Reply:
Because `g_all_hosts` is used to check for OpenShift prerequisites. The main impetus for this PR is the referenced BZ in which openshift-ansible tries to check for packages (in this case, python-docker) on the GlusterFS nodes. If GlusterFS is not native, we should omit them from the set of "all" nodes.

Reply:
...oh yeah, good point!